CBAA_swarm.py

```diff
@@ -152,14 +148,18 @@ def determine_task_list(drone_name: str, drone_info: dict) -> None:
     for j, col in enumerate(bid_columns):
         if debug:
             print("Task {} Bids: {}".format(j + 1, col))
+        best_bid = np.max(col)
```

Comment:
when bids are the same as the previous best, do not change task assignment of agent